doc/api/stream.md

```diff
@@ -28,10 +28,13 @@ objects will rarely (if ever) have need to use the `stream` module directly.
 
 ## Organization of this Document
 
-This document is divided into two primary sections with a third section for
+This document is divided into four primary sections with a fifth section for
 additional notes. The first section explains the elements of the stream API that
 are required to *use* streams within an application. The second section explains
 the elements of the API that are required to *implement* new types of streams.
+The third section is geared towards explanations and *code examples*. The fourth
+section showcases important *npm modules* that make working with streams
```

Comment:
I'm not sure we really want to implement a section like that to be honest. In the past, we've avoided the appearance of promoting certain modules over their competitors. We'd want to be very careful about that.

Reply:
Sounds good! I can get rid of the specific section and maybe instead have examples that use those modules?